### PR TITLE
Kuadrant merge docs

### PR DESCRIPTION
**Pull request by didierofrivia**
This PR brings the documentation that was available in Kuadrant Controller, and adjusts its use for Kuadrant Operator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +71,59 @@ Additionally, Kuadrant provides the following CRDs
...
...<use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,27.654320987654316,0)" xlink:href="#as"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,36.29629629629629,0)" xlink:href="#bl"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,40.61728395061728,0)" xlink:href="#af"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,52.71604938271604,0)" xlink:href="#ah"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,61.35802469135801,0)" xlink:href="#as"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,69.99999999999999,0)" xlink:href="#av"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,78.64197530864196,0)" xlink:href="#ag"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,89.01234567901233,0)" xlink:href="#ai"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,97.6543209876543,0)" xlink:href="#al"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,101.06790123456788,0)" xlink:href="#aJ"/><use transform="matrix(0.04320987654320987,0,0,0.04320987654320987,105.38888888888887,0)" xlink:href="#ar"/></g></defs></g></svg>
```
Comment (eguzki) [inline README.md:74]: is there a way to remove the raw content from the README?? put that in a svg file and make a reference  from the README?

File: doc/development.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Development Guide
+
+## Technology stack required for development
+
+* [operator-sdk] version v1.22.0
+* [kind] version v0.11.1
+* [git][git_tool]
+* [go] version 1.16+
```
Comment (eguzki) [inline doc/development.md:8]: :warning:

File: doc/proposals/rlp-target-gateway-resource.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
+# RLP can target a Gateway resource
```
Comment (eguzki) [inline doc/proposals/rlp-target-gateway-resource.md:1]: Well, this is more than a proposal. This is already what it is implemented. 

Should be included in the docs, not in the proposals.

Comment (eguzki) [reply doc/proposals/rlp-target-gateway-resource.md:1]: maybe we should leave it here for now and remove later in other PRs.

**Review by eguzki — COMMENTED** [review]
looking good

File: README.md [hunk 4]
```diff
@@ -23,10 +71,59 @@ Additionally, Kuadrant provides the following CRDs
...
+
+## Getting started
+
+### Pre-requisites
+
+* Istio is installed in the cluster. Otherwise, refer to the
+  [Istio getting started guide](https://istio.io/latest/docs/setup/getting-started/).
+* Kubernetes Gateway API is installed in the cluster. Otherwise,
+  [configure Istio to expose a service using the Kubernetes Gateway API](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/).
+* Kuadrant is installed in the cluster.
```
Comment (eguzki) [inline README.md:84]: This is "wrong". Kuadrant needs to be installed in the cluster, true, but this guide should show how.

We need a guide to deploy kuadrant. Do we have one or needs to be done? currently it is only creating the kuadrant CR in a given namespace.

I would say the getting started section is:
* Pre-requisites
* Installing kuadrant
* Protect your service
  * If you are an *API Provider*
  * If you are a *Cluster Operator*

Comment (didierofrivia) [reply README.md:84]: You're right, I'll write the missing parts to make more sense

**Review by eguzki — APPROVED** [review]

**Pull request merged**